lib/omnibus/software.rb

```diff
@@ -995,23 +995,25 @@ def source_type
     #
     # @return [true]
     #
-    def build_me
+    # TODO: docco the new arg
+    # TODO: unit test that execute_pre_build and execute_post_build get called correctly
+    def build_me(build_wrappers=[])
```

Comment:
I added this `build_wrapper` concept because it seemed like a good balance of being general enough to use for other things (e.g., we could make the health check incremental to take advantage of the git cache) without doing a lot of refactoring. Thoughts?